ref.yml

```
@@ -0,0 +1,232 @@
+keys:
+- fullname:
+    required: true
+    used:
+    - README.md
+    - index.md
+- shortname:
+    required: true
+    used:
+    - README.md
+    - index.md
+    - coq.opam
+    - extracted.opam
+    - .travis.yml
+    - coq-action.yml
+    - config.yml
+    - default.nix
+- organization:
+    required: true
+    used:
+    - README.md
+    - index.md
+    - coq.opam
+    - extracted.opam
+- travis:
+    type: bool
+    used:
+    - README.md
+- action:
+    type: bool
+    used:
+    - README.md
+- circleci:
+    type: bool
+    used:
+    - README.md
+- community:
+    type: bool
+    used:
+    - README.md
+    - index.md
+- coqdoc:
+    type: bool
+    used:
+    - README.md
+    - coq.opam
+- doi:
+    required: false
+    used:
+    - README.md
+- synopsis:
+    required: true
+    used:
+    - coq.opam
+- description:
+    requied: true
+    used:
+    - README.md
+    - index.md
+    - coq.opam
+- authors:
+    required: true
+    fields:
+    - name:
+        required: true
+    - initial:
+        type: bool
+    - email:
+        required: false
+    used:
+    - README.md
+    - index.md
+    - coq.opam
+    - extracted.opam
+- maintainers:
+    fields:
+    - name:
+        required: true
+    - nickname:
+        required: true
+    used:
+    - README.md
+- license:
+    required: false
+    keys:
+    - fullname:
+        required: true
+    - identifier:
+        required: true
+    - file:
+        required: false
```

I suggest adding the default value:
```suggestion
        required: false
        default: LICENSE
```